eZ/Publish/Core/settings/search_engines/legacy/criterion_handlers_common.yml

```diff
@@ -33,10 +33,10 @@ parameters:
         stopWordThresholdFactor: 0.66
         enableWildcards: true
         commands:
+            - "ascii_search_cleanup"
             - "apostrophe_normalize"
             - "apostrophe_to_doublequote"
             - "ascii_lowercase"
-            - "ascii_search_cleanup"
```

Comment:
> AFAICS this is now fixed for Legacy and Solr engines?

@pspanja: Yes. Actually I think Solr is not affected by this as it has its own `FullText` `CriterionVisitor`. Solr indexes `it's` out-of-the-box just as a single word (AFAIR).
At first I thought `TransformationProcessor` might be common, but now when I look at the code it seems it's only for LegacySE.

@andrerom: if we leave `ascii_search_cleanup` as it was, we'll get more or less the following chain of transformations: 
`it's` -> `it"s` -> `it s`
Then, the test searching for `it's` on LegacySE will also fail (just checked).